src/main/resources/JavaAnnotationfree/mixin.mustache

```diff
@@ -75,9 +75,10 @@ public interface {{classname}}{{#interfaces}}{{#-first}} extends {{/-first}}{{^-
     {{/isInherited}}
 {{/vars}}
 
-    default void mergeFrom({{classname}} target) {
+    default {{classname}} copyTo({{classname}} target) {
```

Comment:
I think the parameter type and return type should instead be `<T extends {{class name}}>`